src/app/profile/update/update.component.html

```diff
@@ -0,0 +1,120 @@
+          <p class="profile-email-wrapper">
+            <span class="fa fa-envelope"></span>
+            <a href="javascript:void(0)" class="margin-left-5 margin-right-5"
+               (click)="setElementFocus($event, _email)">Add an email</a>
+          </p>
+          <p>
+            <span class="fa fa-external-link"></span>
+            <a href="javascript:void(0)" class="margin-left-5 margin-right-5"
+               (click)="setElementFocus($event, _email)">Add a url</a>
+          </p>
+          <a href="javascript:void(0)" class="margin-left-5 margin-right-5"
+             (click)="setElementFocus($event, _email)">Add a bio</a>
+        </div>
+      </div>
+    </div>
+    <!-- center column -->
+    <div class="col-md-8">
+      <form class="col-sm-8" role="form">
+        <p class="fields-status-pf">The fields marked with <span class="required-pf">*</span> are required.</p>
+        <h2>Your profile</h2>
+        <div class="form-group">
+          <label for="username" class="control-label required-pf">Username</label>
+          <div [ngClass]="{'has-error': usernameInvalid === true}">
+            <input type="text" class="form-control" id="username" name="username" required
+                   [(ngModel)]="username">
+            <span class="help-block">
+              Username must be less than 63 characters and can't begin with a dash or underscore. They may contain
+              lowercase letters, numbers, dashes, or underscores.
+            </span>
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="name" class="control-label">Name</label>
+          <div [ngClass]="{'has-error': nameInvalid === true}">
+            <input type="text" class="form-control" id="name" name="name"
+                   [(ngModel)]="name">
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="hideName" class="sr-only">Hide name</label>
+          <div class="checkbox">
+            <input type="checkbox" id="hideName" name="hideName"
+                   [(ngModel)]="hideName">Don't show my name
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="name" class="control-label">Email</label>
+          <div [ngClass]="{'has-error': emailInvalid === true}">
+            <input type="text" class="form-control" id="email" name="email" #_email
+                   [(ngModel)]="email">
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="hideName" class="sr-only">Hide email</label>
+          <div class="checkbox">
+            <input type="checkbox" id="hideEmail" name="hideEmail"
+                   [(ngModel)]="hideEmail">Don't show this email
+          </div>
+        </div>
+        <h2 class="margin-top-40">Connected accounts</h2>
+        <p class="getting-started">
+          This is where we explain that the user must connet their GitHub and OpenShift accounts to OpenShit.io. We also
```

Comment:
This text should be updated to whatever the getting-started page text gets changed to.